CHANGELOG.md

```diff
@@ -2,6 +2,112 @@
+- vertica: Deprecates the `ddtrace.contrib.vertica` module. Use attributes exposed in `ddtrace.contrib.vertica.__all__` instead.
+- webbrowser: Deprecates the `ddtrace.contrib.webbrowser` module. Use attributes exposed in `ddtrace.contrib.webbrowser.__all__` instead.
+- wsgi: Deprecates the `ddtrace.contrib.wsgi` module. Use attributes exposed in `ddtrace.contrib.wsgi.__all__` instead.
+- yaaredis: Deprecates all modules in the `ddtrace.contrib.yaaredis` package. Use attributes exposed in `ddtrace.contrib.yaaredis.__all__` instead.
+### Bug Fixes
+
+- CI Visibility: This fix resolves an issue where exceptions other than timeouts and connection errors raised while fetching the list of skippable tests for ITR were not being handled correctly and caused the tracer to crash.
+- CI Visibility: Fixes a bug where `.git` was incorrectly being stripped from repository URLs when extracting service names, resulting in `g`, `i`, or `t` being removed (eg: `test-environment.git` incorrectly becoming `test-environmen`)
+- LLM Observability: This fix resolves an issue where custom trace filters were being overwritten in forked processes.
+- Fix for a side-effect issue with module import callbacks that could cause a runtime exception.
+- llmobs: This fix resolves an issue where session_id was being defaulted to trace_id which was causing unexpected UI behavior.- LLM Observability: This fix resolves an issue where LLM Observability spans were not being submitted in forked processes, such as when using `celery` or `gunicorn` workers. The LLM Observability writer thread now automatically restarts when a forked process is detected.
+- Fixed an issue with some module imports with native specs that don't support attribute assignments, resulting in a `TypeError` exception at runtime.
+- This fix resolves an issue where `ddtrace` package files were published with incorrect file attributes.
+- tracer: This fix resolves an issue where django db instrumentation could fail.
+- openai: Fixes a bug where \`asyncio.TimeoutError\`s were not being propagated correctly from canceled OpenAI API requests.
+
+- tracing: Adds `ddtrace.trace.Context` to the public api. This class can now be used to propagate context across execution boundaries (ex: threads).
+- aiobotocore: Fix an issue where the <span class="title-ref">\_make_api_call</span> arguments were not captured correctly when using keyword arguments.
+- tracing(django): This fix resolves a bug where ddtrace was exhausting a Django stream response before returning it to user.
+- LLM Observability: Fixes an issue in the OpenAI integration where integration metrics would still be submitted even if `LLMObs.enable(agentless_enabled=True)` was set.
+- internal: Fix for `Already mutably borrowed` error when rate limiter is accessed across threads.
+- internal: Fix `Already mutably borrowed` error by reverting back to pure-python rate limiter.
```

Comment:
```suggestion
- internal: Fixes the `Already mutably borrowed` error by reverting back to pure-python rate limiter.
```

Reply:
Same comment here, do we need this for an internal fix? Or is the category incorrect?

Reply:
this is correct, it is a bug that impacted customers, but the bug was located in the "internal" scope.